modules/rest-api/pages/rest-configure-ldap.adoc

```diff
@@ -122,4 +122,12 @@ Value must be an integer between 1 and 100: the default is 10.
 | cacheValueLifetime
 | Lifetime of values in cache in milliseconds. Default 300000 ms.
 
+| clientTLSCert
+| Path to a certificate to be used by Couchbase Server as a client, in authentication with LDAP.
+The `serverCertValidation` flag must be set to `true`, and a value must be specified for the `clientTLSkey` flag.
```

Comment:
This is not correct. `serverCertValidation` has nothing to do with client cert authentication.